app/views/phases/_submissions_table.html.erb

```diff
@@ -6,34 +6,37 @@
   <%= render 'shared/alert_error', alert_heading: t('alerts.recused_evaluator.heading'), alert_text: t('alerts.recused_evaluator.text') %>
 <% end %>
 
-<table class="usa-table usa-table--stacked-header usa-table--borderless width-full">
+<%= render partial: "phases/sort_and_filter" %>
+
+<table class="usa-table usa-table--stacked-header usa-table--borderless width-full gray-header">
     <thead>
       <tr>
         <th scope="col">Submission ID</th>
         <th scope="col">Eligible for Evaluation</th>
         <th scope="col">Selected to Advance</th>
         <th scope="col">Assigned Evaluators</th>
         <th scope="col">Average Score</th>
+        <th scope="col"><span class="usa-sr-only">View Submission</span></th>
       </tr>
     </thead>
     <tbody>
       <% @submissions.each do |submission| %>
           <tr>
             <th data-label="Submission ID" scope="row">
-              <span class="mobile-lg:display-none">Submission ID </span><%= submission.id %>
+              <span class="mobile-lg:display-none">Submission ID</span><%= submission.id %>
```

Comment:
![Screenshot 2024-12-10 at 4 16 16 PM](https://github.com/user-attachments/assets/14d30a90-337e-44fd-afe5-e37e3c4a097d)

The mobile layout is missing the space between the label and id, should revert this
```suggestion
              <span class="mobile-lg:display-none">Submission ID </span><%= submission.id %>
```